---
layout: workshop_main_3day
permalink: /rnaseq_2017
title: RNA-Seq
header1: Workshop Pages for Students
header2: Informatics for RNA-Seq Analysis
image: /site_images/CBW_RNA_seq_icon.jpg
---
# Welcome <a id="welcome"></a>

Welcome to Informatics for RNA-Seq Analysis 2017.  

The course schedule can be found [here](https://bioinformaticsdotca.github.io/rnaseq_2017_schedule).

Pre-readings, pre-work, and laptop setup instructions can be found [here](https://bioinformaticsdotca.github.io/rnaseq_2017_prework)

If you have questions or comments, visit the workshop [Q&A forum](https://noteapp.com/RNA2017).

***

# Class Photo
 
<img src="https://github.com/bioinformaticsdotca/RNASeq_2017/blob/master/CBW-July-10.jpg?raw=true" alt="Class Photo" width="750" />


# Day 1 <a id="day1"></a>

**Day 1 and Day 2 refer to content in the [RNA-Seq wiki](https://github.com/griffithlab/rnaseq_tutorial/wiki).**

##  Welcome 

  *<font color="#827e9c">Ann Meyer</font>* 

[Lecture](https://bioinformatics.ca/rnaseq-2017-mod0)  

## Introduction to Cloud Computing

*<font color="#827e9c">Obi Griffith</font>* 

[Lecture](https://bioinformatics.ca/rnaseq-2017-modcc)

* We have set up 30 instances on the Amazon cloud - one for each student. In order to log in to your instance, you will need a security certificate. If you plan on using Linux or Mac OS X, please download [this certificate](http://cbwmain.dyndns.info/private/CBWNY.pem). Otherwise if you plan on using Windows (with Putty), please download [this certificate](http://cbwmain.dyndns.info/private/CBWNY.ppk).

* Detail instructions can be found [here](https://bioinformaticsdotca.github.io/AWS_setup).

## Module 1: Introduction to RNA Sequencing Analysis

*<font color="#827e9c">Malachi Griffith</font>* 

[Lecture](https://bioinformatics.ca/rnaseq-2017-mod1)

### Lab Practical

[Lab practical](https://bioinformatics.ca/rnaseq-2017-mod1b)

  [Installation](https://github.com/griffithlab/rnaseq_tutorial/wiki/Installation)
  
  [Reference Genomes](https://github.com/griffithlab/rnaseq_tutorial/wiki/Reference-Genome)
  
  [Annotations](https://github.com/griffithlab/rnaseq_tutorial/wiki/Annotation)
  
  [Indexing](https://github.com/griffithlab/rnaseq_tutorial/wiki/Indexing)
  
  [Data](https://github.com/griffithlab/rnaseq_tutorial/wiki/RNAseq-Data)
  
  [Data QC](https://github.com/griffithlab/rnaseq_tutorial/wiki/PreAlignment-QC)


## Module 2: RNA-seq Alignment and Visualization

*<font color="#827e9c">Fouad Yousif</font>* 

[Lecture](https://bioinformatics.ca/rnaseq-2017-mod2)

### Lab Practical

[Lab Practical](https://bioinformatics.ca/rnaseq-2017-mod2b)

  [Adapter Trim](https://github.com/griffithlab/rnaseq_tutorial/wiki/Adapter-Trim)
  
  [Alignment](https://github.com/griffithlab/rnaseq_tutorial/wiki/Alignment)
  
  [IGV](https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial)

  [Alignment Visualization](https://github.com/griffithlab/rnaseq_tutorial/wiki/PostAlignment-Visualization)
  
  [Alignment QC](https://github.com/griffithlab/rnaseq_tutorial/wiki/PostAlignment-QC)

## Integrated Assignment

*<font color="827e9c">Fouad Yousif</font>*

Paper: [Recurrent chimeric RNAs enriched in human prostate cancer identified by deep sequencing](http://www.ncbi.nlm.nih.gov/pubmed/21571633)

[Assignment Questions](https://github.com/griffithlab/rnaseq_tutorial/wiki/Integrated-Assignment)

[Assignment Answers](https://github.com/griffithlab/rnaseq_tutorial/wiki/Integrated-Assignment-Answers)  


***

# Day 2 <a id="day2"></a>

## Module 3: Expression and Differential Expression

*<font color="#827e9c">Obi Griffith</font>* 

[Lecture](https://bioinformatics.ca/rnaseq-2017-mod3)

### Lab Practical

[Lab Practical](https://bioinformatics.ca/rnaseq-2017-mod3b)

 [Expression](https://github.com/griffithlab/rnaseq_tutorial/wiki/Expression)
 
 [Differential Expression](https://github.com/griffithlab/rnaseq_tutorial/wiki/Differential-Expression)
 
 [DE Visualization](https://github.com/griffithlab/rnaseq_tutorial/wiki/DE-Visualization)

## Module 4: Reference Free Alignment

*<font color="#827e9c">Malachi Griffith</font>* 

[Kallisto](https://github.com/griffithlab/rnaseq_tutorial/wiki/Kallisto)  

## Module 5: Isoform Discovery and Alternative Expression

*<font color="#827e9c">Malachi Griffith</font>* 

[Lecture](https://bioinformatics.ca/rnaseq-2017-mod5)

### Lab Practical

[Lab Practical](https://bioinformatics.ca/rnaseq-2017-mod5b)

  [Ref Guided](https://github.com/griffithlab/rnaseq_tutorial/wiki/Reference-Guided-Transcript-Assembly)
  
  [De novo](https://github.com/griffithlab/rnaseq_tutorial/wiki/de-novo-Transcript-Assembly)
  
  [Merging](https://github.com/griffithlab/rnaseq_tutorial/wiki/Transcript-Assembly-Merge)
  
  [Differential Splicing](https://github.com/griffithlab/rnaseq_tutorial/wiki/Differential-Splicing)
  
  [Splicing Visualization](https://github.com/griffithlab/rnaseq_tutorial/wiki/Transcript-Assembly-Visualization)

## Integrated Assignment

[Assignment Questions](https://github.com/griffithlab/rnaseq_tutorial/wiki/Integrated-Assignment)  

[Assignment Answers](https://github.com/griffithlab/rnaseq_tutorial/wiki/Integrated-Assignment-Answers)  

## Keeping Up-to-date with RNA-Seq Analysis Developments

For additional resources, tutorials, future directions, and more please refer to the [RNA-seq wiki](http://www.rnaseq.wiki/)

***

# Day 3 <a id="day3"></a>

## Module 6: Genome-Free De Novo Transcriptome Assembly

*<font color="#827e9c">Brian Haas</font>* 

[Lecture](https://bioinformatics.ca/rnaseq-2017-mod6)  

[Bandage](https://rrwick.github.io/Bandage/)  

[Lab practical](https://bioinformaticsdotca.github.io/rnaseq_2017_tutorial6)  


## Module 7: Functional Annotation and Analysis of Transcripts

*<font color="#827e9c">Brian Haas</font>* 

[Lecture](https://bioinformatics.ca/rnaseq-2017-mod7)  

[Lab practical](https://bioinformaticsdotca.github.io/rnaseq_2017_tutorial7)  

***

Thank you for attending the Informatics and RNA-Seq Analysis workshop! Help us make this workshop better by filling out [our survey](https://goo.gl/forms/u6Ifq7KdegnpFXHx1)
